.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    name: Node.js ${{ matrix.node_version }} - ${{ matrix.architecture }} on ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        node_version:
+          # - 12
+          - 14
```

Could you add Node 16 (latest non-LTS release) as well? I don't want future surprises like this to come up. I also would prefer to keep Node 12 in there [as it's in maintenance mode until April 30th of next year](https://nodejs.org/en/about/releases/).

```suggestion
          - 12
          - 14
          - 16
```

---

We now use 14, 16 and 18